src/redist/targets/GenerateLayout.targets

```diff
@@ -163,6 +152,13 @@
         <DownloadFileName>$(DownloadedNetStandardTargetingPackInstallerFileName)</DownloadFileName>
         <AccessToken>$(CoreSetupBlobAccessTokenParam)</AccessToken>
       </BundledInstallerComponent>
+
+      <BundledInstallerComponent Include="DownloadedNetStandardTargetingPackInstallerFile"
```

Comment:
Looks like the Include name is wrong, copied from above.

Reply:
Fixed.